pythran/config.py

```diff
@@ -69,8 +70,8 @@ def parse_define(define):
 
     # blas dependency
     numpy_blas = numpy_sys.get_info("blas")
-    extension['libraries'].extend(numpy_blas['libraries'])
-    extension['library_dirs'].extend(numpy_blas['library_dirs'])
+    extension['libraries'].extend(numpy_blas.get('libraries', []))
+    extension['library_dirs'].extend(numpy_blas.get('library_dirs', []))
```

Comment:
I will try to solve it. Once others things are OK, I will try to find a procedure to install blas on windows and use it.